### DISCUSSION TITLE
Netbeans learning course

### DISCUSSION THREAD

**Discussion by Biulk** [apache/netbeans#4934] [General]
### Description

I am following the learning path but I found that the links that refer to support files do not work. This can tell me where to find them.
https://netbeans.apache.org//kb/docs/java-se.html
Thank you

### Related issues

no

### Are you willing to submit a pull request?

Yes

### Code of Conduct

Yes

**Comment by ebarboni** [comment]
Hi @Biulk could you be more precise on missing file? 

Since former netbeans site shutdown we have a lot of them missing

**Comment by neilcsmith-net** [comment]
Moving this to discussions as we have with other issues opened about the website.  Be good to change or remove links.
